lib/put.lua

```diff
@@ -6,19 +6,22 @@ local current_time = redis.call('TIME')
 local current_timestamp_ms = current_time[1] * 1000 + current_time[2] / 1000
 
 local current = redis.call('HMGET', KEYS[1], 'r')
+local new_content
 
 if current[1] then
-  tokens_to_add = math.min(current[1] + tokens_to_add, bucket_size)
+  new_content = math.min(current[1] + tokens_to_add, bucket_size)
+else
+  new_content = math.min(bucket_size + tokens_to_add, bucket_size)
```

Comment:
Minor: I think intention would be clearer if we say that if there's no current, then current is bucket_size, and then just do `new_content = math.min(current[1] + tokens_to_add, bucket_size)`

Reply:
Addressed ✔️ 
Thanks for the suggestion!